EmpleoDotNet.sln

```diff
@@ -1,17 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30110.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.23107.0
```

Comment:
I think this now forces everybody to use VS2015. Ye all warned to upgrade now =D